backend/database/seeders/PermissionSeeder.php

```diff
@@ -198,16 +198,30 @@ public function run()
                 'slug' => 'delete-tag',
                 'description' => 'Can delete a tag',
             ],
+            [
+                'id' => 27,
+                'name' => 'View',
+                'category' => 'User',
+                'slug' => 'view-user',
+                'description' => 'Can view users',
+            ],
+            [
+                'id' => 28,
+                'name' => 'View All',
+                'category' => 'Teams',
+                'slug' => 'viewAll-teams',
```

Comment:
```suggestion
                'slug' => 'view-all-teams',
```

Reply:
it would break a function i think since i factored that slugs only have 1 `-`